joomla/joomla-cms
# remove extra Tinymce code

Pull Request for Issue # 9339.
#### Summary of Changes

Remove duplicated code icon
#### Testing Instructions

Check in the advanced AND extended modes of tinmyce and you will see two code icons. One after the image icon and one at the very end

This PR removes the second instance
